### PR TITLE
feat(automation): add GH release updates

### PR DESCRIPTION
**Pull request by georgettica**
- Dockerfile was changed to make verifying what changed in the release easier
- sorted github releases in dockerfile
- added a short explanation on each binary we add

### CODE REVIEW

**Comment by georgettica** [comment]
see #104 

this PR removes the github-actions

**Comment by georgettica** [comment]
the slug is mostly used for me,  as I have the GHA that shows the changed line, so I can copy the slug and search google for what is new in that release (instead of partially copying the URL

**Comment by georgettica** [comment]
tested and compiles :)

File: Dockerfile [hunk 1]
```diff
@@ -80,36 +80,51 @@ RUN curl -sSlo epel-gpg https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8
...
+ARG ROSA_VERSION="tags/v1.1.11"
+ENV ROSA_URL_SLUG="openshift/rosa"
+ENV ROSA_URL="https://api.github.com/repos/${ROSA_URL_SLUG}/releases/${ROSA_VERSION}"
 
+# Add `velero` utility for quick backup verification
 # Replace "/latest" with "/tags/{tag}" to pin to a specific version (eg: "/tags/v0.4.0")
-ARG VELERO_VERSION="tags/v1.7.2"
-ENV VELERO_URL="https://api.github.com/repos/vmware-tanzu/velero/releases/${VELERO_VERSION}"
+# the URL_SLUG is for checking the releasenotes when a version updates
+ARG VELERO_VERSION="tags/v1.8.1"
```
Comment (cblecker) [inline Dockerfile:112]: We run 1.7 not 1.8 in OSD

Comment (georgettica) [reply Dockerfile:112]: Aren't the versions backwards compatible?

And shouldn't we upgrade then?

Anyways will downgrade :)

**Pull request merged**